manifests/dellhw_exporter.pp

```diff
@@ -86,7 +86,7 @@
   }
 
   $real_omreport_path = "--collectors-omreport=${omreport_path}"
-  $real_scrape_port = "--web-listen-address=${scrape_port}"
+  $real_scrape_port = "--web-listen-address=:${scrape_port}"
```

Comment:
I'm wondering if people abused the `scrape_port` parameter to set also an ip address. with your new syntax this won't work anymore. Could you introduce another parameter for the ip address?

Reply:
Hi,

Yes at first that's how I tried to fix the problem temporarily (by passing scrape_port => ':9137', or  scrape_port => scrape_port => '127.0.0.1:9137') 
But the scrape_port parameter type is **Stdlib::Port** so its not possible.

Do you still want a dedicated ip address parameter ?

Reply:
oh right, it's already the port type. I will merge this PR as is. But it would be awesome if you could add an ip address parameter in another PR.

Reply:
Thanks. 

Ok I will make an other PR for the IP address.